.goreleaser.yml

```diff
@@ -0,0 +1,109 @@
+project_name: vogelkop
+before:
+  hooks:
+    - go mod download
+
+builds:
+  -
+    id: go
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+      - arm
+    goarm:
+      - 6
+      - 7
+    ldflags:
+      - -s -w
+      - -X github.com/metal-toolbox/version.appName={{.ProjectName}}
+      - -X github.com/metal-toolbox/version.version={{.Version}}
+      - -X github.com/metal-toolbox/version.commit={{.Commit}}
+      - -X github.com/metal-toolbox/version.date={{.Date}}
+      - -X github.com/metal-toolbox/version.builtBy=goreleaser
+
+checksum:
+  name_template: 'checksums.txt'
+
+snapshot:
+  name_template: "{{ .Tag }}-next"
+
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - '^docs:'
+      - '^test:'
+
+dockers:
+  -
+    use: buildx
+    goos: linux
+    goarch: amd64
+    image_templates:
+    - "ghcr.io/metal-toolbox/{{.ProjectName}}:{{ .Tag }}-amd64"
+    - "ghcr.io/metal-toolbox/{{.ProjectName}}:latest-amd64"
+    build_flag_templates:
+      - "--platform=linux/amd64"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+  -
+    use: buildx
+    goos: linux
+    goarch: arm64
+    image_templates:
+    - "ghcr.io/metal-toolbox/{{.ProjectName}}:{{ .Tag }}-arm64v8"
+    - "ghcr.io/metal-toolbox/{{.ProjectName}}:latest-arm64v8"
+    build_flag_templates:
+      - "--platform=linux/arm64/v8"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+
+docker_manifests:
+  # https://goreleaser.com/customization/docker_manifest/
+  - name_template: ghcr.io/metal-toolbox/{{.ProjectName}}:{{ .Tag }}
```

Comment:
kind of annoying that some of the templates are tight vs padded. `{{.Tag}}` probably to stay in line with all the other uses here.